src/DB_Command.php

```diff
@@ -204,11 +204,17 @@ public function clean( $_, $assoc_args ) {
 	 *     Success: Database checked.
 	 */
 	public function check( $_, $assoc_args ) {
-		self::run( Utils\esc_cmd( '/usr/bin/env mysqlcheck --no-defaults %s', DB_NAME ), array_merge( $assoc_args, array(
-			'check' => true,
-		) ) );
+		self::run(
+			Utils\esc_cmd( '/usr/bin/env mysqlcheck --no-defaults %s', DB_NAME ),
+			array_merge(
```

Comment:
The `array_merge()` here is total overkill.
Can you rewrite this in the following way:
```php
$assoc_args['check'] = true;
self::run(
	Utils\esc_cmd( '/usr/bin/env mysqlcheck --no-defaults %s', DB_NAME ),
	$assoc_args,
);
```